.github/ISSUE_TEMPLATE/add_plugin.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Add plugin
+about: Request to add your plugin to the Marketplace
+title: "Add $REPOSITORY_NAME to Marketplace"
+labels: Plugin/New
+assignees: hanzei
+
+---
+<!--
+Thank you very for submitting your Plugin! It will go through a review process to make sure it follow the quality standard  of the Marketplace. This process might take a couple of weeks bepending on how many changes are needed.
+Read https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/ before submitting your plugin.
+-->
+
+#### Summary
+<!--
+A brief description what your plugin does.
+-->
+
+#### Review Commit
+<!--
+Please link the commit or release that should be used for review.
+-->
+
+## Checklist
+<!--
+Please go trough this checklist and confirm every item. If your plugin doesn't fulfil every item, leave a comment explaining why and if you will fix this.
+-->
+
+**Product Requirements**
```

Comment:
If there are any updates on https://github.com/mattermost/mattermost-developer-documentation/pull/524, I will update this list.

It's a bit annoying to sync these two on every change, but I think it's worth.